libswift/Sources/SIL/Builder.swift

```diff
@@ -55,16 +55,27 @@ public struct Builder {
   public func createCondFail(condition: Value, message: String) -> CondFailInst {
     notifyInstructionsChanged()
     return message.withBridgedStringRef { messageStr in
-      let cf = SILBuilder_createCondFail(
-        bridgedInsPoint, location.bridgedLocation, condition.bridged, messageStr)
-      return cf.getAs(CondFailInst.self)
+      var builder = SILBuilder_init(bridgedInsPoint)
+      let cf = builder.createCondFail(
+        unsafeBitCast(location, to: swift.SILLocation.self),
+        unsafeBitCast(condition.bridged, to: swift.SILValue.self),
+        unsafeBitCast(messageStr, to: llvm.StringRef.self),
+        /*inverted*/false)
+      return Unmanaged<Instruction>.fromOpaque(UnsafeRawPointer(cf)!)
```

Comment:
Is it possible to extract the boilerplate of `Unmanaged<Instruction>.fromOpaque(UnsafeRawPointer(i)!).takeUnretainedValue()  as! InstType` into a utility function?

Reply:
If that's the case, I'd assume `unsafeBitCast` boilerplate can be avoided somehow as well?